_posts/news/2015-07-03-announcing-scalajs-0.6.4.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: Announcing Scala.js 0.6.4
+category: news
+tags: [releases]
+---
+{% include JB/setup %}
+
+We are excited to announce the release of Scala.js 0.6.4!
+
+This release brings support for Scala 2.11.7 and 2.12.0-M1, as well as a significant part of the Java collections library in `java.util`, thanks to [@andreaTP](https://github.com/andreaTP) and [@nicolasstucki](https://github.com/nicolasstucki).
+It also fixes numerous bugs.
+
+## Getting started
+
+If you are new to Scala.js, head over to
+[the tutorial]({{ BASE_PATH }}/doc/tutorial.html).
+
+## Release notes
+
+For changes in the 0.6.x series compared to 0.5.x, read [the announcement of 0.6.0]({{ BASE_PATH }}/news/2015/02/05/announcing-scalajs-0.6.0/).
+
+As a minor release, 0.6.4 is backward source and binary compatible with previous releases in the 0.6.x series.
+Libraries compiled with earlier versions can be used with 0.6.4 without change.
+However, it is not forward compatible: libraries compiled with 0.6.4 cannot be used by projects using 0.6.{0-3}.
+
+Please report any issues [on GitHub](https://github.com/scala-js/scala-js/issues).
+
+## Improvements
+
+### Java Collections API
+
+Some Scala libraries use the collections of Java in some cases.
+To help port these libraries to Scala.js, a number of collection types have been ported.
+At the moment, the following data structures are supported:
+
+* `List`: `ArrayList`, `LinkedList`, `CopyOnWriteArrayList`
+* `Set`: `HashSet`, `LinkedHashSet`, `ConcurrentSkipListSet`
+* `Map`: `HashMap`, `LinkedHashMap`, `ConcurrentHashMap`
+* `Queue`: `LinkedList`, `ConcurrentLinkedQueue`
+
```

Comment:
Why? They completely respect their semantics. The simplifications are an implementation detail.